src/main/java/com/commercehub/gradle/plugin/avro/GenerateAvroJavaTask.java

```diff
@@ -39,6 +46,7 @@ public void setEncoding(String encoding) {
 
     @Input
     public String getStringType() {
+        System.out.println("StringType was get: " + stringType);
```

Comment:
No need to fix that.  I'll handle it in a round of minor touch-ups I was going to do anyway.